analyses/run-gistic/gistic-example.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+set -e 
+set -o pipefail
+
+# Configure environmental variables for MCR
+export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/opt/mcr/v83/runtime/glnxa64:/opt/mcr/v83/bin/glnxa64:/opt/mcr/v83/sys/os/glnxa64:
```

Comment:
```suggestion
export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/opt/mcr/v83/runtime/glnxa64:/opt/mcr/v83/bin/glnxa64:/opt/mcr/v83/sys/os/glnxa64
```
doubt you need this last colon since you're not going to append anything after this

Reply:
(I realize that one was from me back in the previous PR 😁 )